### PR TITLE
Add SelexION separation voltage

### PR DESCRIPTION
**Pull request by edeutsch**
Add new term SelexION separation voltage based on documentation here:
https://sciex.com/content/dam/SCIEX/pdf/tech-notes/all/SelexION-Technology-Solution-to-Selectivity-Challenges.pdf

### CODE REVIEW

**Comment by mobiusklein** [comment]
Is this distinct from `SelexION compensation voltage`? https://www.ebi.ac.uk/ols4/ontologies/ms/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMS_1003371?lang=en

**Comment by edeutsch** [comment]
I think yes, based on the documentation here: https://sciex.com/content/dam/SCIEX/pdf/tech-notes/all/SelexION-Technology-Solution-to-Selectivity-Challenges.pdf

**Comment by edeutsch** [comment]
@mobiusklein I can't merge without an approval. Would you approve if you're happy with this?

**Review by mobiusklein — APPROVED** [review]

**Pull request merged**
